internal/clients/nightfall/nightfall.go

```diff
@@ -87,7 +84,7 @@ func blurContent(content string) string {
 }
 
 func getCommentMsg(finding *nf.Finding) string {
-	if finding.Finding == "" || finding.Detector.DisplayName == "" {
+	if (finding.Finding == "" && finding.RedactedFinding == "") || finding.Detector.DisplayName == "" {
```

Comment:
If we are redacting the finding, then do we still need to blur the content with the custom `blurContent` function?